docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
+)
+t.track(page_view)
 ```
+## Creating a `PagePing` Event
+Track engagement with a web page over time, via a `PagePing` event. Each ping represents a single heartbeat.
 
-The `event_json` is represented using the SelfDescribingJson class. It has two fields: `schema` and `data`. `data` is a dictionary containing the properties of the unstructured event. `schema` identifies the JSON schema against which `data` should be validated. This schema should be available in your [Iglu schema registry](https://github.com/snowplow/iglu) and your Snowplow pipeline configured so that that that registry is included in your Iglu resolver.
-
-For more on JSON schema, see the [blog post](http://snowplowanalytics.com/blog/2014/05/15/introducing-self-describing-jsons/).
-
-Many Snowplow users use the above method to track _all_ their events i.e. only record event types that they have defined. However, there are a number of "out of the box" events that have dedicated tracking methods. These are detailed below:
-
-### Track page view
-
-Use `track_page_view()` to track a user viewing a page within your app or website. The arguments are:
-
-| **Argument** | **Description** | **Required?** | ****Type**** |
+| **Property** | **Description** | **Type** | **Required?** |
 | --- | --- | --- | --- |
-| `page_url` | The URL of the page | Yes | Non-empty string |
-| `page_title` | The title of the page | No | String |
-| `referrer` | The address which linked to the page | No | String |
-| `context` | Custom context for the event | No | List(SelfDescribingJson) |
-| `tstamp` | When the pageview occurred | No | Positive integer |
-| `event_subject` (since v0.9.0) | The subject for the event | No | Subject |
+| `page_url` | URL of the viewed page | `string` | Yes |
+| `page_title` | Title of the viewed page | `string` | No |
+| `referrer` | The address which linked to the page | `string` | No |
+| `min_x` | Minimum page x offset seen in the last ping period | `int` | No |
+| `max_x` | Maximum page x offset seen in the last ping period | `int` | No |
+| `min_y` | Minimum page y offset seen in the last ping period | `int` | No |
+| `max_y` | Maximum page y offset seen in the last ping period | `int` | No |
+| `event_subject` | The subject for the event | `Subject` | No |
+| `context` | Custom context for the event | `List(SelfDescribingJson)` | No |
+| `true_timestamp` | When the page ping occurred | `int | float` | No |
 
 Example:
-
 ```python
-t.track_page_view("www.example.com", "example", "www.referrer.com")
+page_ping = PagePing(
+  page_url="https://www.snowplow.io",
+  page_title="Homepage",
+  referrer="https://docs.snowplow.io/docs",
+)
+t.track(page_ping)
 ```
 
-### Track page pings
+## Creating a `ScreenView` Event
 
-Use `track_page_ping()` to track engagement with a web page over time, via a heartbeat event. (Each ping represents a single heartbeat.)
+Use the `ScreenView` to track a user viewing a screen (or equivalent) within your app.
```

Comment:
When should I use screen view and when should I use page view?

Reply:
Screenview is the app equivalent of pageview for the web, I've clarified in the pageview section and the table summary